environment.yml

```diff
@@ -0,0 +1,19 @@
+name: linsolve
```

Comment:
This might be nit-picky but I would appreciate if the name was the same as the environment being created in conda.

Reply:
Woops. Fixed in 2b82e0daa1652123ebe93d842f422875cb27cefb